File: infer/tests/codetoanalyze/java/classloads/Basic.java
/*
 * Copyright (c) 2018-present, Facebook, Inc.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

class Basic {
  void foo_report(Other o) {
    o.bar();
  }

  void baz_no_report(Another a) {
    a.baz();
  }
}

class Other {
  void bar() {}
}

class Another {
  void baz() {}
}

File: infer/tests/codetoanalyze/c/sil/basic.c
/*
 * Copyright (c) Facebook, Inc. and its affiliates.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

int add(int x, int y) { return x + y; }

int local_vars(int a) {
  int b = a + 1;
  int c = b * 2;
  return c;
}
